source/common/filter/http/filter_config_discovery_impl.h

```diff
@@ -121,6 +129,7 @@ class FilterConfigSubscription
   absl::optional<Envoy::Http::FilterFactoryCb> last_config_{absl::nullopt};
   std::string last_type_url_;
   std::string last_version_info_;
+  envoy::config::core::v3::TypedExtensionConfig last_filter_config_;
```

Comment:
I think we intentionally avoid storing the entire filter config, can we just track whether the last filter passed the terminal filter check?

Reply:
Yes, we only need `it_terminal_filter`